python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -134,12 +145,17 @@ class GrapheneAssetNode(graphene.ObjectType):
         partitions=graphene.List(graphene.String),
     )
     materializationCountByPartition = non_null_list(GrapheneMaterializationCount)
+    materializationCountByDimensionPartition = graphene.Field(
```

Comment:
Fundamentally, does this need to have a different query implementation from the above?

Should the value in the `partition` column be mirrored as a tag, or should they be completely disjoint?  I think we could potentially set it as a tag, with a different key.